docs/reference/commandline/start.md

```diff
@@ -30,7 +30,15 @@ To start a container, specify the container ID or image name. For example:
 $ docker start my_container
 ```
 
-### Start a container and attach to its STDOUT/STDERR
+### <a name="attach"></a>  Start a container and attach to its STDOUT/STDERR (--attach, -a)
+
+> This option is experimental.
+>
+> This command is experimental on the Docker daemon. It should not be used in
+> production environments.
+> To enable experimental features on the Docker daemon, edit the
+> [daemon.json](/engine/reference/commandline/dockerd/
+#daemon-configuration-file)
 
```

Comment:
Oh! A bit too eager; this one is not experimental (only the checkpoint/restore parts are)